Vagrantfile

```diff
@@ -245,6 +246,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       os.ssh_username = user = voc['ssh_user']|| creds['OSSshUser']  || "root"           # login for the VM instance
       os.server_name  = ENV['OS_HOSTNAME']    || vagrant_openshift_config['instance_name'] # name for the instance created
       full_provision(override.vm, user)
+      floating_ip_pool = creds['OSFloatingIPPool'] || ENV['OS_FLOATING_IP_POOL']
```

Comment:
@danmcp said:
Do you mind collapsing these next 6 lines into 2?
@akostadinov said:
Possible but would 2 two very long lines. We need to have `:auto` as a sym and `false` as value, not string to get desired behavior. If you confirm I can make it 2 lines but would be hardly readable. Like:
```
os.floating_ip = (creds['OSFloatingIPPool'] || ENV['OS_FLOATING_IP_POOL']) == ":auto" ? :auto : (creds['OSFloatingIPPool'] || ENV['OS_FLOATING_IP_POOL'])
```
What  would you suggest?